### PR TITLE
Fix/dependabot problems

### PR DESCRIPTION
**Pull request by manuerumx**
**Method:**
Dependabot automatically updates dependencies defined in the project

**Expected Behavior:**
Pull requests with updates are generated.

**Actual Result:**
Dependabot cannot find some packages and fails to generate updates.

**Cause:**
Still unknown, the problem is present in other projects, but we discover findalab was using other branches rather the defined in the composer.json file but we expect this could fix that issue.
The project was using different versions of PHP

**Solution:**

- Standardize the project to use at least PHP 7.0*
- Fix the composer.lock file and update to use the expected branches.

The changes break the Behat suites. The PR includes a [temporary fix](https://github.com/Medology/findalab/pull/208/files#diff-38f01125e159afa7bd888512c79ed2c4R265) but is required in another issue to update the project and use the latest version of [FlexibleMink](https://github.com/Medology/FlexibleMink/)

> The original function exists on FlexibleMink, but the version FindALab is using apparently includes a bug in the commented method. `$this->assertNodeElementVisibleInViewport($button);` 

**Customer Impact (for #releasenotes):**
None

**QA Notes (How to set up the system for manual testing)**
Changes not testable. The changes only affect the testing part.

For #204

### CODE REVIEW

File: features/bootstrap/WebContext.php [hunk 1]
```diff
@@ -252,6 +257,28 @@ public function setGeoLocation($x, $y)
             'success(position);}');
     }
 
+    /**
+     * @ToDo Remove this function after FlexibleMink 2 is updated.
+     *
+     * {@inheritdoc}
+     */
+    public function pressButton($locator, TraversableElement $context = null)
```
Comment (manuerumx) [inline features/bootstrap/WebContext.php:265]: The original function exists on FlexibleMink, but the version FindALab uses apparently includes a bug in the commented method. `$this->assertNodeElementVisibleInViewport($button);`

File: docker/lib/images.sh [hunk 2]
```diff
@@ -20,5 +20,12 @@ export SELENIUM_REPO=standalone-chrome-debug
 export SELENIUM_TAG=3.14.0-dubnium
 export SELENIUM_IMAGE=${SELENIUM_HOST}/${SELENIUM_ACCOUNT}/${SELENIUM_REPO}:${SELENIUM_TAG}
 
+
+export PHP_HOST=registry.hub.docker.com
+export PHP_OWNER=chekote
+export PHP_REPO=php
+export PHP_TAG=7.0.33.c-laravel5.2
```
Comment (jose-ochoa) [inline docker/lib/images.sh:27]: Shouldn't this be just 7.0.33, this isn't a laravel project

File: features/bootstrap/WebContext.php [hunk 3]
```diff
@@ -60,6 +61,10 @@ public function gatherContexts(BeforeScenarioScope $scope)
      */
     public function initWindowSize()
     {
+        if (!$this->getSession()->isStarted()) {
```
Comment (jose-ochoa) [inline features/bootstrap/WebContext.php:64]: This should be in its own method being called with `@BeforeScenario`

File: docker/lib/images.sh [hunk 4]
```diff
@@ -20,5 +20,12 @@ export SELENIUM_REPO=standalone-chrome-debug
 export SELENIUM_TAG=3.14.0-dubnium
 export SELENIUM_IMAGE=${SELENIUM_HOST}/${SELENIUM_ACCOUNT}/${SELENIUM_REPO}:${SELENIUM_TAG}
 
+
+export PHP_HOST=registry.hub.docker.com
```
Comment (jose-ochoa) [inline docker/lib/images.sh:24]: Please remove this and  create `export DOCKER_REGISTRY_HOST=registry.hub.docker.com` at the top of the file, so we can start eliminating the repetitiveness.

File: docker/lib/images.sh (hunk 2 above)
Comment (manuerumx) [reply docker/lib/images.sh:27]: I tried but fails to start due to missing dependencies, Findalab is a Lumen project 🤦 , and Donald doesn't have specific images for Lumen.

File: features/bootstrap/WebContext.php (hunk 3 above)
Comment (manuerumx) [reply features/bootstrap/WebContext.php:64]: Done, but I pair with Donald yesterday and he explains to me there's no guarantee, that method is going to run before the `initWindowSize()` due to how Behat load the contexts. So, I include a call in this method.

File: features/bootstrap/WebContext.php [hunk 5]
```diff
@@ -60,6 +73,7 @@ public function gatherContexts(BeforeScenarioScope $scope)
      */
     public function initWindowSize()
     {
+        $this->initSession();
```
Comment (Chekote) [inline features/bootstrap/WebContext.php:76]: You should not both call this in this method AND have `initSession()` annotated with `@BeforeScenario`. If you want to try and get the order of operations correct, then you would not manually invoke it here and instead rely on `@BeforeScenario`. If you do not want to fight with Behat's inability to define the order in which each `@BeforeScenario` will fire, then you should remove `@BeforeScenario` and leave the call here.

File: features/bootstrap/WebContext.php [hunk 6]
```diff
@@ -252,6 +266,28 @@ public function setGeoLocation($x, $y)
...
+            return $this->scrollToButton($locator, $context);
+        });
+
+        $this->waitFor(function () use ($button, $locator) {
+            if ($button->getAttribute('disabled') === 'disabled') {
+                throw new ExpectationException("Unable to press disabled button '$locator'.", $this->getSession());
+            }
+        });
+
+        //$this->assertNodeElementVisibleInViewport($button);
```
Comment (Chekote) [inline features/bootstrap/WebContext.php:287]: Please remove this line, and instead add a description to this method's doc block explaining why we are overriding the method from FlexibleMink.

**Pull request merged**